v-next/hardhat-network-helpers/README.md

```diff
@@ -2,9 +2,38 @@
 
 Hardhat Network Helpers is a library that provides a set of utility functions to interact with the [Hardhat Network](https://hardhat.org/hardhat-network/docs).
```

Comment:
Should we call it plugin for consistency or stick with library?

```suggestion
Hardhat Network Helpers is a plugin that provides a set of utility functions to interact with the [Hardhat Network](https://hardhat.org/hardhat-network/docs).
```

Reply:
This is true! It's now a plugin